tool/make-native-distribution.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+# This script creates a distribution of TruffleRuby and Sulong
+# compiled to a native image by Substrate VM.
+
+# Do not use jt.rb here to minimze dependencies and avoid compatibility problems
+# between the current commit and the built tag's jt.rb.
+
+set -e
+set -x
+
+# Tag to use for the different repositories
+RELEASE=0.32
+TAG=vm-enterprise-$RELEASE
+
+# In which directory to create the release, will contain the final archive
+PREFIX="../release"
+
+# Check platform
+case $(uname) in
+  Linux)
+    os=linux
+    ;;
+  Darwin)
+    os=darwin
+    ;;
+  *)
+    echo "unknown platform $(uname)" 1>&2
+    exit 1
+    ;;
+esac
+
+# Check architecture
+case $(uname -m) in
+  x86_64)
+    arch=amd64
+    ;;
+  *)
+    echo "unknown architecture $(uname -m)" 1>&2
+    exit 1
+    ;;
+esac
+
+revision=$(git rev-parse --short HEAD)
+
+rm -rf "${PREFIX:?}"/*
+mkdir -p "$PREFIX"
+
+# Expand $PREFIX
+PREFIX=$(cd "$PREFIX" && pwd -P)
+
+mkdir -p "$PREFIX/build"
+cd "$PREFIX/build"
+
+git clone --branch $TAG "$(mx urlrewrite https://github.com/oracle/truffleruby.git)"
+git clone --branch $TAG "$(mx urlrewrite https://github.com/graalvm/sulong.git)"
+git clone --branch $TAG "$(mx urlrewrite https://github.com/oracle/graal.git)"
+
+cd truffleruby
+
+build_home=$(pwd -P)
+
+release_home="$PREFIX/truffleruby"
+mkdir -p "$release_home"
+
+# Use our own GEM_HOME when building
+export TRUFFLERUBY_RESILIENT_GEM_HOME=true
+
+# Build
+
+# Build Sulong first so that TRUFFLERUBY-ZIP includes sulong-libs
```

Comment:
This is just about putting together a tarball with all the files isn't it? It's not some problem with our Sulong dependency?

Reply:
Yes, it is.
This is a way to get the libsulong.bc and libsulong.so in the tarball (`jt build native sulong` also does a similar thing).
Depending on Sulong should help to make this cleaner and part of just "mx build", but that's for later. In 0.32, we didn't depend on Sulong directly.